### ISSUE TITLE
Issues with durable sink after upgrade to current stable version

### ISSUE THREAD

**Issue by JanKnipp** [serilog/serilog-sinks-elasticsearch#66]
Hi,
I've just upgraded from an older version (of serilog and the elasticsearch sink) and whenever i try to use the durable sink I get an "System.MissingMethodException" in "Serilog.Sinks.Elasticsearch.dll"

Method not found: "Void Serilog.Sinks.RollingFile.RollingFileSink..ctor(System.String, Serilog.Formatting.ITextFormatter, System.Nullable`1<Int64>, System.Nullable`1<Int32>, System.Text.Encoding, Boolean)".

As soon as i remove the BufferBaseFilename from the elasticsearchOptions i do no longer get an exception - but of course i'm loosing the durability :(

` var elasticSearchOptions = new ElasticsearchSinkOptions(new Uri("http://localhost:9200"))
                                           {
                                               AutoRegisterTemplate = true,
                                               IndexFormat = "some-app-{0:yyyy.MM.dd}",
                                               BufferBaseFilename = @"e:\some-app-buffer.log",
                                               BatchPostingLimit = 10000,
                                               MinimumLogEventLevel = LogEventLevel.Information
                                           };
            Logger = new LoggerConfiguration().WriteTo.Elasticsearch(elasticSearchOptions).CreateLogger();`

Regards,
Jan

**Comment by daniel-meismer-zocdoc** [comment]
I just had this problem too. I was able to get it working by downgrading `Serilog.Sinks.RollingFile` to version `2.2.0`. This is okay *for me* since I'm using Serilog for the first time in a PoC, though I don't know what the consequences (if any) would be in production.  

My guess is an optional parameter was added to the `RollingFileSink` constructor in `v3.x`, which unfortunately breaks binary compatibility.
